fantasi/Tuura/Fantasi/Main.hs

```diff
@@ -2,9 +2,8 @@ module Tuura.Fantasi.Main (main) where
 
 import Tuura.Fantasi.Options
 import qualified Pangraph.GraphML.Parser as P
-import qualified Pangraph.VHDL.EnvironmentWriter as W1
-import qualified Pangraph.VHDL.GraphWriter as W2
-import Data.ByteString.Char8 (pack)
+import qualified Pangraph.VHDL as V
```

Comment:
Let's import as `VHDL`. If you see a `V.foo` in the code you'll need to guess what it means and/or look at the list of imports, whereas `VHDL.foo` is pretty self-explanatory.